README.md

```diff
@@ -1,27 +1,50 @@
 # Memory Inspector
 
-This extension contributes a set of widgets for viewing memory in different ways.
+A Visual Studio Code extension that provides a powerful and configurable memory viewer that works with debug adapters.
 
-## Requirements
+![Screenshot of the Memory Inspector](https://raw.githubusercontent.com/eclipse-cdt-cloud/vscode-memory-inspector/master/media/memory-inspector-screenshot.png)
 
-This extension must be used in conjunction with a Debug Adapter that implements a ReadMemoryRequest handler or alternative custom request that returns
-memory data. It has been tested against the [CDT-GDB Adapter](https://github.com/eclipse-cdt/cdt-gdb-adapter) used as the backend for the 
-[CDT-GDB VSCode](https://github.com/eclipse-cdt/cdt-gdb-vscode) plugin.
+## Features
 
-## The Memory Provider
+- **Multiple Format Display**: Shows memory data in different formats, provided your debug adapter supports the `ReadMemoryRequest`.
+- **Address Navigation**: Easily jump to and scroll through memory addresses.
+- **Variable Highlights**: Marks variable memory ranges.
+- **Edit Memory**: Allows in-place memory editing, if the debug adapter supports the `WriteMemoryRequest`.
+- **Memory Management**: Enables saving and restoring memory data for specific address ranges.
+- **Custom Views**: Create and customize as many memory views as you need.
+- **Lock Views**: Keep views static, unaffected by updates from the debug session.
+- And much more
+
+## Getting Started
+
+1. **Install**: Add the extension to VS Code.
+2. **Debug Session**: Start with a debug adapter that has the [`ReadMemory` request](https://microsoft.github.io/debug-adapter-protocol/specification#Requests_ReadMemory) capability.
```

Comment:
We should point users to the global `Debug Types` setting to enable this for any adapter that supports the `ReadMemory` request.

![image](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/assets/24247651/459016a1-7718-46c2-8aca-b9fc095727a6)